src/pages/settings/Payments/AddDebitCardPage.js

```diff
@@ -218,6 +219,7 @@ class DebitCardPage extends Component {
                                         label={this.props.translate('addDebitCardPage.cvv')}
                                         onChangeText={securityCode => this.clearErrorAndSetValue('securityCode', securityCode)}
                                         value={this.state.securityCode}
+                                        maxLength="4"
```

Comment:
Also, CVV in the Uk is only 3 digits. Is there a case where cvv can be 4 digits?

Reply:
Ah the American Express cards use 4 digits for ccv

Reply:
Cool, thanks for confirming!